src/subresource-signing.js

```diff
@@ -39,8 +54,10 @@ var srs = {
         /**
          * Assume the contents is valid, and attach the script
          * to the document so it runs.
+         *
+         * @param contents
          */
-        attach_script: function srs_attach_script(contents) {
+        attach_script: function(contents) {
```

Comment:
I thought it would be on the one who produced the most programming constructs?! Ok fine...

[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) + Readability > Stack traces 

Readability since it's an uncommon syntax and confused me at first as well. Also justifying stack traces since at this level since the only item changed here prone to failure is not ... well it's just not something that needs a named function for stack traces if you're attaching a script to the DOM.